instance/github.py

```diff
@@ -158,13 +159,8 @@ def __init__(self, number, source_fork_name, target_fork_name, branch_name, titl
         self.title = title
         self.username = username
         self.body = body
-
-    @property
-    def extra_settings(self):
-        """
-        Extra settings contained in the PR body
-        """
-        return get_settings_from_pr_body(self.body)
+        self.extra_settings = get_settings_from_pr_body(body)
+        self.extra_settings_dict = yaml.load(self.extra_settings) or {}
```

Comment:
Can we avoid duplicating the data? That requires to keep `extra_settings_dict` manually in sync with `extra_settings`, which itself has to be kept in sync with `pr_body`. Duplicating data has a tendency to add lots of subtle bugs on the long term. We should make sure we have our data structure right, if we feel the need to duplicate it it might be indicative of an underlying problem - I'd rather have the data set once, and then computed from that single place by the rest of the code.